module4/src/Test2.java

```diff
@@ -0,0 +1,31 @@
+public class Test2 {
+
+    private int firstVariable;
+    private int secondVariable;
+
+    public Test2() {
+        this.firstVariable = 111;
+        this.secondVariable = 222;
+    }
```

Comment:
You changed the values 6 and 8 to 111 and 222. Has anything changed dramatically?